pkgs/applications/audio/airwindows/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, requireFile, unzip, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "airwindows-${version}";
```

Comment:
`name` is automatically constructed from `pname`-`version`.
```suggestion
  pname = "airwindows";
```

Reply:
Done.

Reply:
@OPNA2608 I seem to have messed up the push, sorry.
I'm working on getting the new fork to build, and once will than push the fixes.

Reply:
I had some trouble pushing to this branch, so I made https://github.com/NixOS/nixpkgs/pull/123954